pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -226,39 +178,6 @@ def reduce(self, output, group: Optional[Any] = None, reduce_op: Optional[Union[
 
         return output
 
-    def post_dispatch(self) -> None:
```

Comment:
You still need to reload the  weights and results from rank 0 to main process.

Reply:
@tchaton As it is a subclass of `DDPSpawn`, the methods are available.

https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/plugins/training_type/ddp_spawn.py#L159

Reply:
Yes, and the mq_queue gets seem in the right order. All good !